Task/FP-1550-custom-text-in-admin-ui (#467)
TACC/Core-CMS: README.md

<sub>To learn more, see [Static Files](#static-files).</sub>

### Customizing Text in Admin UI

1. Create file `/taccsite_cms/locale/en/LC_MESSAGES/django.po`.

> 2. Add this content to the file for just the strings to translate.

Reviewer: ```suggestion
2. To the file add the strings to translate (and the appropriate comments).
```
